types/web3/utils.d.ts

```diff
@@ -2,21 +2,26 @@ import BigNumber = require("bn.js");
 import * as us from "underscore";
 
 type Unit =
+    | "noether"
+    | "wei"
     | "kwei"
-    | "femtoether"
+    | "Kwei"
     | "babbage"
+    | "femtoether"
     | "mwei"
-    | "picoether"
+    | "Mwei"
     | "lovelace"
-    | "qwei"
-    | "nanoether"
+    | "picoether"
+    | "gwei"
+    | "Gwei"
     | "shannon"
-    | "microether"
-    | "szabo"
+    | "nanoether"
     | "nano"
+    | "szabo"
+    | "microether"
     | "micro"
-    | "milliether"
     | "finney"
+    | "milliether"
```

Comment:
For reference https://web3js.readthedocs.io/en/1.0/web3-utils.html?highlight=toWei#id76
List goes not alphabetically but by the number of trailing zeros.